Algorithms/check_permutation.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# CtCI 6th Edition Problem 1.2
+# Given two strings, write a method to decide if one is a permutation of the
+# other.
+
+from timeit import timeit
+import collections
+
+
+def is_permutation(string1, string2):
+    return (len(string1) == len(string2)
+            and sorted(string1) == sorted(string2))
```

Comment:
1. I'm following what is defined in PEP-8 for  binary operators, this would seem to apply to `and` also: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator. In any case I think there reasoning is sound to include it after the line break.

2.) PEP-8 limits lines to 80 characters, without the carriage return the line would be slightly over.